src/context/reducers/dataReducer.js

```diff
@@ -44,6 +44,31 @@ const userReducer = (state, action) => {
         vid: { ...state.vid, error: payload, loading: false, items: [] },
       };
 
+    case videoConstants.FILTER_CATEGORY:
+      console.log("category", payload);
```

Comment:
can remove this console.log

Reply:
Fixed - [d73cfa5](https://github.com/SukanyaSen530/DrawFlix/pull/4/commits/d73cfa5f65d1c25829c21559a914bf0445b14948)